pkgs/development/libraries/libgssglue/default.nix

```
@@ -1,14 +1,32 @@
-{ lib, stdenv, fetchurl, libkrb5 }:
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  autoconf,
+  automake,
+  libtool,
+  libkrb5,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libgssglue";
-  version = "0.4";
+  version = "0.9";
 
-  src = fetchurl {
-    url = "http://www.citi.umich.edu/projects/nfsv4/linux/libgssglue/${pname}-${version}.tar.gz";
-    sha256 = "0fh475kxzlabwz30wz3bf7i8kfqiqzhfahayx3jj79rba1sily9z";
+  src = fetchFromGitLab {
+    owner = "gsasl";
+    repo = "libgssglue";
+    rev = "tags/debian/${finalAttrs.version}";
+    hash = "sha256-D+Us4wk0Fh4Idco8HKcF1EStcYyWWoUuXRshAQAgJtk=";
   };
 
+  preConfigure = "./bootstrap";
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    libtool
+  ];
```

Did not test this but in most cases this should work.

```suggestion
  nativeBuildInputs = [
    autoreconfHook
  ];
```

---

Seems to fail with `autoreconf: error: automake failed with exit status: 1`

---

Seems to fail:
```
configure.ac:13: installing 'build-aux/compile'
configure.ac:14: installing 'build-aux/config.guess'
configure.ac:14: installing 'build-aux/config.sub'
configure.ac:8: installing 'build-aux/install-sh'
configure.ac:8: installing 'build-aux/missing'
Makefile.am: error: required file './ChangeLog' not found
Makefile.am: installing './INSTALL'
src/Makefile.am: installing 'build-aux/depcomp'
parallel-tests: installing 'build-aux/test-driver'
autoreconf: error: automake failed with exit status: 1
```

---

looks like their bootstrap also creates a ChangeLog file instead of just executing autoreconf,
 can be fixed with setting `preConfigure = "touch ChangeLog";`